tests/test_models.py

```diff
@@ -150,6 +150,26 @@ def test_consent_not_stored_audit_available_externally_managed(self):
     def test_consent_not_stored_no_audit_available(self):
         assert self.enrollment.consent_available() is False
 
+    @ddt.data(
+        ('at_login', None, 1, 'not_set'),
+        ('at_login', True, 1, 'enabled'),
+        ('at_login', False, 1, 'disabled'),
+        ('externally_managed', None, 1, 'external'),
+        ('externally_managed', False, 1, 'external'),
+        ('externally_managed', True, 1, 'external'),
+    )
+    @ddt.unpack
+    def test_create_related_consent_audit(self, requirement, this_state, expected_count, expected_value):
```

Comment:
@haikuginger Looks like `expected_count` is always `1` -- can we get rid of this parameter (and the `if` in L170 below)?